doc/developer/webservices.rst

```diff
@@ -240,3 +289,26 @@ Result
 
 The 'Content-Type' header is 'text/csv',
 and the data contains the given 'csv' data.
+
+Geometry operations
+===================
+
+This service gives geometry operations (actually only one)
+
+Difference
+----------
+
+URL: ``.../difference``
+
+Method: ``POST``
+
+Data:
+
+.. code:: json
+
+   {
+       "geom": <The geom to extrude>
+       "diff": <The geom to extrude with>
```

Comment:
Finally I think that the different names are so bad that I can have something like this:
``` json
{
    arguments: [<geomA>, <geomB>]
}
```
What do you think ?